tests/test_noop.py

```diff
@@ -0,0 +1,32 @@
+"""noop tests"""
+
+
+from __future__ import print_function
+import pytest
+import wandb
+
+
+def test_noop():
+    run = wandb.init(config={"foo": {"bar": {"x": "y"}}}, mode="disabled")
+    wandb.log({"x": {"y": "z"}})
+    wandb.log({"foo": {"bar": {"x": "y"}}})
+    assert wandb.run == run
+    assert wandb.config == run.config
+    assert wandb.summary == run.summary
+    assert run.config.foo["bar"]["x"] == "y"
+    assert wandb.summary["x"].y == "z"
+    assert wandb.summary["foo"].bar.x == "y"
+    wandb.summary.foo["bar"].update({"a": "b"})
+    assert wandb.summary.foo.bar.a == "b"
+
+
+def test_bad_url():
```

Comment:
I think you need the test settings fixture to turn off console logging and reset the settings instance...